docs/SelfHostedRunnerSetup.md

```diff
@@ -1,26 +1,39 @@
-6) Obtain an [authentication token](https://github.com/microsoft/ebpf-for-windows/settings/actions/runners/new). This requires administrator permissions in the project.
-7) Configure action runner.
-   1) ```./config.cmd --url https://github.com/microsoft/ebpf-for-windows --token <action runner token>```
-8) Change action runner service to run as "LocalSystem".
-   1) Open services.msc.
-   2) Locate "GitHub Action Runner ...".
-   3) Right Click -> Properties -> Log On.
-   4) Change "Log on as:" to "Local System Account".
-9) Install (for all users) the Prerequisites listed at the top of the [Getting Started page](./GettingStarted.md#prerequisites)
-10) Reboot the runner.
+3) Obtain an [authentication token](https://github.com/microsoft/ebpf-for-windows/settings/actions/runners/new). This requires administrator permissions in the project.
+4) Configure action runner as follows:
+   ```./config.cmd --url https://github.com/microsoft/ebpf-for-windows --labels 'kernel_test_vm' --token <action runner token> --runasservice --windowslogonaccount <account> --windowslogonpassword <password> ```
+   The `--runasservice` parameter makes the action runner run as a Windows service. The runner service runs as
+   `NetworkService` by default. However, the `Kernel_Test_VM` workflow performs operations on a test VM that requires
+   administrator privilege. So, the credentials of an account with administrator privilege must be supplied in
+   `windowslogonaccount` and `windowslogonpassword` parameters.
+5) Install the build tools as follows:
+   1) Install [Git for Windows 64-bit](https://git-scm.com/download/win).
+   2) Install [Visual Studio Build Tools 2019](https://aka.ms/vs/16/release/vs_buildtools.exe).
+   3) From the Visual Studio Installer GUI choose the following:
+      1) MSVC v142 - VS 2019 C++ x64/x86 build tools (Latest).
+      2) Windows 10 SDK. Choose the latest available version.
+      3) C++ CMake tools for Windows.
+      4) MSVC v142 - VS 2019 C++ x64/x86 Spectre Mitigated libs (Latest).
+   4) [Windows Driver Kit for Windows 10, version 2004](https://go.microsoft.com/fwlink/?linkid=2128854). The WDK
+     version is 10.0.19041.685. Care must be taken that the version of the Windows 10 SDK in step 5.3.2 matches the
+     WDK version. If the build fails with `error MSB8020: The build tools for WindowsKernelModeDriver10.0 cannot
+     be found` then perform the following mitigation steps:
+      1) Copy `c:\Program Files (x86)\Windows Kits\10\Vsix\VS2019\WDK.vsix` to a file with the extension .zip instead of .vsix and extract it.
+      2) Copy all files under `$MSBuild\Microsoft\*` to `c:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Microsoft`
+   5) Download [Nuget.exe](https://www.nuget.org/downloads). Do not copy the nuget.exe file in Windows\System32 folder. Instead copy it in some other folder and make sure that the path to that folder is added to the PATH system environmental variable.
```

Comment:
Why can't it be in System32?

Reply:
Added a line why; also see https://github.com/NuGet/Home/issues/6459